editor/scene_tree_dock.cpp

```diff
@@ -3103,8 +3103,32 @@ bool SceneTreeDock::_check_node_recursive(Variant &r_variant, Node *p_node, Node
 void SceneTreeDock::set_edited_scene(Node *p_scene) {
 	edited_scene = p_scene;
 }
-
+bool is_same_selection(const Vector<Node *> &p_first, const List<Node *> &p_second) {
```

Comment:
```suggestion
static bool _is_same_selection(const Vector<Node *> &p_first, const List<Node *> &p_second) {
```
Use static method to avoid pollution of namespace